pkgs/by-name/un/unit/package.nix

```diff
@@ -8,9 +8,7 @@
   withPython3 ? true,
   python3,
   ncurses,
-  withPHP81 ? true,
-  php81,
-  withPHP82 ? false,
+  withPHP82 ? true,
```

Comment:
@Izorkin fyi. I only did the bare minimum to make it work without PHP 8.1.

If this only supports PHP 8.2, I'll drop the package next year. Please make sure that it supports all available PHP interpreters at some point.